packages/vaults/src/manager/vault_manager.ts

```diff
@@ -60,26 +102,49 @@ export class VaultManager implements VaultManagerI {
         return transactions
     }
 
+    /**
+     * Retrieves the balance of the canister.
```

Comment:
Let's be specific about the balance. Cycles?

Reply:
Yeap. Good catch